tests/pages/pmmInventoryPage.js

```diff
@@ -160,6 +163,10 @@ module.exports = {
     return serviceIdLocator;
   },
 
+  async getNodeName(nodeID) {
+    return await I.grabTextFrom(`//table//tr/td[2][contains(text(),"${nodeID}")]/following-sibling::td[2]`);
```

Comment:
you could get this using API too

Reply:
It's not used. I forgot to delete. Thanks! :)